vizro-core/docs/pages/user-guides/custom-charts.md

```diff
@@ -3,26 +3,26 @@
 This guide shows you how to create custom charts and how to add them to your dashboard.
 The [`Graph`][vizro.models.Graph] model accepts the `figure` argument, where you can enter _any_ [`plotly.express`](https://plotly.com/python/plotly-express/) chart as explained in the [user guide on graphs](graph.md).
 
-## Overview of custom charts
-
-In general, the usage of the custom chart decorator `@capture("graph")` is required if your plotly chart requires any post-update calls or customization.
-
 ### When to use a custom chart
+In general, the usage of the custom chart decorator `@capture("graph")` is required if your plotly chart requires any post-update calls or customization e.g:
```

Comment:
```suggestion
In general, you should use the custom chart decorator `@capture("graph")` if your plotly chart needs any post-update calls or customization e.g:
```

Reply:
Is Vale running OK? I would have expected an error for the e.g. here. When you've successfully merged this PR, next sprint when I am back on Vizro tickets, I'll take a look to find out why it didn't.

Reply:
Interesting - yes, there are no errors raised or at least it's passing CI!